packages/ui/.env.development

```diff
@@ -0,0 +1 @@
+IS_NETWORK_MAINNET="true";
```

Comment:
environment files are not meant to be checked into version control as they usually hold sensitive information or are system specific.

Reply:
yes, my bad. I am now using the file `.env.local` which is already added to gitignore